Doc/library/ctypes.rst

```diff
@@ -2018,13 +2018,13 @@ Utility functions
    Does the same as the C ``sizeof`` operator.
 
 
-.. function:: string_at(address, size=-1)
+.. function:: string_at(ptr, size=-1)
 
-   This function returns the C string starting at memory address *address* as a bytes
+   This function returns the C string starting at memory address *ptr* as a bytes
```

Comment:
There were some suggestions about rewording this (and the wstring_at docstring below) in the bpo discussion:
https://bugs.python.org/issue43803?@ok_message=issue%2043803%20versions%20edited%20ok&@template=item#msg390795

Reply:
You mean adding additional clarification to mention that it can accept parameters of types mentioned in https://bugs.python.org/issue43803#msg390795 ?

By the way, thanks for the quick review!

Reply:
No I mean the paragraph that begins with "If the docstring is change to use "ptr"" and the following one.

Reply:
Addressed with eb21757855544aef960c92546da35ae66ec0279c